lib/workers/repository/update/branch/index.ts

```diff
@@ -198,15 +198,21 @@ export async function processBranch(
             });
             const newBodyHash = hashBody(newBody);
             if (newBodyHash !== branchPr.bodyStruct?.hash) {
```

Comment:
`newBodyHash !== branchPr.bodyStruct?.hash` will be true on the first dry run when the branch is modified because we adding a `rebasingNotice`. therefore the PR gets updated.